x-pack/plugins/actions/server/routes/connector/get_all/transforms/transform_connectors_response/v1.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FindConnectorResult } from '../../../../../application/connector/types';
+import { ConnectorResponse } from '../../../../../../common/routes/connector/response';
+import { RewriteResponseCase } from '../../../../../../common/rewrite_request_case';
+
+export const transformGetAllConnectorsResponse: RewriteResponseCase<FindConnectorResult[]> = (
+  results
+): ConnectorResponse[] => {
+  return results.map(
+    ({
+      actionTypeId,
+      isPreconfigured,
+      isDeprecated,
+      referencedByCount,
+      isMissingSecrets,
+      isSystemAction,
+      ...res
+    }) => ({
+      ...res,
```

Comment:
Ideally, we should avoid spreading for transforms, especially for versioned transforms to avoid properties accidentally getting into response types. I think it would be better if we were more specific about the properties.